articles/introduction-to-influxdb/index.md

```diff
@@ -0,0 +1,123 @@
+DBMS or database management systems play a key role as the foundational building blocks of any modern application. Whether you are a database administrator, a developer, or an IT professional, extensive knowledge of databases is essential. In this article, we will look at a class of databases called time-series databases and understand how they work. We are also going to explore the basics of the most popular time-series database called influx DB. By the end of this article, you will have influx DB installed on your system and be able to execute custom queries. 
+  
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [Time Series Database](#time-series-database)
+- [InfluxDB](#influxdb)
+- [Installing InfluxDB](#installing-influxdb)
+- [Running InfluxDB](#running-influxdb)
+- [Executing Queries](#executing-queries)
+- [Conclusion](#conclusion)
+  
+### Prerequisites
+- Basic understanding of [database management systems](https://www.tutorialspoint.com/dbms/index.htm)
+- Basic understanding of [Python](https://www.python.org/about/gettingstarted/).
+
+### Time Series Database
+According to the [official documentation](https://www.influxdata.com/time-series-database/), A time-series database (TSDB) is a database optimized for timestamped or time-series data. Time series data are simply measurements or events that are tracked, monitored, downsampled, and aggregated over time. Furthermore, time-series databases are built to specifically handle this type of data with a timestamp associated with it. 
+
+Let us consider a few examples to understand this better. The stock market is a very volatile and time-dependent system. It is essential to monitor and store this information along with the timestamp of when the transactions happened. Another example would be gathering sensor data on temperature, pressure, or humidity. These data types require a time associated with each record to help us understand the "trend" or analyze a pattern that changes over time. However, one might ask: Why can't we just add a column/field called time and keep track of the timestamp when the event occurred. This is possible, but there are two ways to do this: When we get a new record or data for a particular value such as temperature, should we overwrite the previous reading, or should we create a new row in the database? Both methods will provide the current or most recent values. However, we can only analyze the trend or the system changes over a period of time if we insert a new row. This is what time-series databases are optimized to do.
```

Comment:
However, we can only analyze the trend or the system changes? is it the trend of the system changes? If not, the last sentence needs minor restructuring.